### ISSUE TITLE
no-commented-out-code is not detecting errors in jest tests

### ISSUE THREAD

**Issue by wattry** [bahmutov/eslint-rules#37]
I am trying to ensure commented out code is not submit in any of our projects.

I have the following .eslintrc and I am using a jest plugin so that errors are not thrown for functions that don't exist.

```
{
 {
  "env": {
    "es6": true,
    "browser": false,
    "node": true
  },
  "extends": [
    "eslint:recommended",
    "airbnb",
    "plugin:prettier/recommended",
    "plugin:jest/recommended"
  ],
  "plugins": ["extra-rules",
    "jest"
  ],
  "parserOptions": {
    "ecmaVersion": 2018
  },
  "rules": {
    "prettier/prettier": "error",
    "no-console": "off",
    "extra-rules/no-commented-out-code": "error",
    "no-use-before-define": ["error", {
      "functions": false
    }]
  }
}

```

When I comment out code in my rootdir/app*.js it detects the issues. However, if I comment out code in my tests located at rootdir/test/*.js no issues are output.

I expect the issues to be submit on all js files.

**Comment by jarrietas** [comment]
Happening also when using a mocha like structure: describe ... it ...
